conn.go

```diff
@@ -648,7 +643,7 @@ func (c *Conn) handleQueuedPackets(ctx context.Context) error {
 				return e
 			}
 		} else if err != nil {
-			return e
+			return err
```

Comment:
I think this is the same situation as on L610, the additional checking for `alertError` doesn't do anything. But if this exists on master still then lets fix that there instead.

Reply:
I think so. I didn't want to change any behavior so went for the minimal change. 

On line 610 the caller checks whether alertError is fatal so we dont need the check here and just return err if it isn't nil. 

The caller for line 651 doesn't check if the alertError is fatal so we check it here and suppress alertErrors that are not fatal.